packages/reward-root-submitter/tests/resources/reward_output/rewardProgramID=0xA2e8225dE0385ebC20B3C0160864f4e20a750cfd/paymentCycle=24000000/parameters.json

```diff
@@ -0,0 +1 @@
+{"core": {"subgraph_config_location": "s3://tall-data-dev/subgraph_extraction/staging_rewards/0.0.1/", "reward_program_id": "0xA2e8225dE0385ebC20B3C0160864f4e20a750cfd", "docker_image": "reward_programs", "payment_cycle_length": 32768, "valid_from": 24000000, "valid_to": 26000000}, "user_defined": {"reward_program_type": "usage", "parameters": {"token": "0xcd7AB5c678Bc0b90dD6f870B8F214c10A943FC67", "base_reward": 100, "transaction_factor": 6, "spend_factor": 3, "valid_duration": 500000}}, "run": {"payment_cycle": 24000000}}
```

Comment:
I've been trying to just remove "reward_program_id" from the "core" parameters. Think an input of the scheduler.

Reply:
Something like this https://github.com/cardstack/cardstack/blob/c8254863d39143182c54026f1f118dca814b4522/packages/cardpay-reward-programs/input/parameters.json

Reply:
Yeah that's fine, probably needs to go in the run params? In this case, I'm not actually doing anything with those files, I'd had the reward programs write out the exact param file they used alongside the results, so these just happen to be there. I might remove them from the test data.

Reply:
I think provisionally lets include them inside "run" params.